controllers/storagecluster/storageclasses.go

```diff
@@ -228,6 +228,12 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 		disable:           managementSpec.DisableStorageClass,
 		isClusterExternal: initData.Spec.ExternalStorage.Enable,
 	}
+	if dataPool != "" {
```

Comment:
We are checking whether the `dataPool` argument is empty or not, but where are we using it?
In the line#233, we are directly creating a value (using the function `generateNameForEcCephBlockPool()`) for the `dataPool` key.

Reply:
That's a good point. Where is `dataPool` used? With EC, the SC config looks like this:

https://github.com/rook/rook/blob/fffb86295600196389de81905c3402f9aec6e751/deploy/examples/csi/rbd/storageclass-ec.yaml#L45-L46